students/Cracolici_Jon/lesson08/Circle.py

```
@@ -0,0 +1,138 @@
+#Jon Cracolici
+#UW-Python Cert
+#Lesson 08 - Creating a Circle Class w/Tests
+
+
+import math
+
+class Circle:
+    """This is my Circle class. It constructs a circle using the radius, or the diameter. """
+    def __init__(self, radius):
+        """This is the initializer, which also checks the input of radius. Task 1"""
+        try:
+            radius = float(radius)
+            if (radius < 0.0) == True:
+                print('The radius must be a non-negative numeric value.')
+                return None
```

you should raise an exception here instead of returning None